### DISCUSSION TITLE
Is there "rejected" kind of event?

### DISCUSSION THREAD

**Discussion by uhfath** [uNmAnNeR/imaskjs#683] [Ideas]
When a user types something that is not accepted by the mask it would be a nice UX to inform that it's an invalid input (with some hint under control).
Is anything like this currently possible?
There are `accept` and `complete` events but no `reject` or something like that.
For example, if we have a mask for a phone number, when a user presses a letter key the input element will not react and in this case I wanted to show some hint that the key he pressed is not supported in this field.

**Comment by uNmAnNeR** [comment]
nope
